package.json

```diff
@@ -9,23 +9,23 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
-    "@okta/okta-auth-js": "^4.9.2",
-    "@okta/okta-vue": "^4.0.1",
+    "@okta/okta-auth-js": "^5.1.1",
+    "@okta/okta-vue": "^5.0.0",
     "axios": "^0.21.1",
     "core-js": "^3.15.1",
     "semantic-ui-css": "^2.4.1",
     "text-encoding": "^0.7.0",
-    "vue": "^3.1.1",
+    "vue": "^3.1.2",
     "vue-router": "^4.0.10"
   },
   "devDependencies": {
     "@vue/cli-plugin-babel": "~4.5.13",
     "@vue/cli-plugin-eslint": "~4.5.13",
     "@vue/cli-plugin-router": "~4.5.13",
     "@vue/cli-service": "~4.5.13",
-    "@vue/compiler-sfc": "^3.1.1",
+    "@vue/compiler-sfc": "^3.1.2",
     "babel-eslint": "^10.1.0",
-    "eslint": "^7.32.0",
+    "eslint": "^6.7.2",
```

Comment:
@mraible eslint v7 didn't work for me, is this expected?

Reply:
It works for me:

```
$ npm run lint

> okta-vue-sample@1.0.0 lint /Users/mraible/dev/okta-samples/okta-vue-sample
> vue-cli-service lint

 DONE  No lint errors found!
```

What fails for you?

Reply:
ugg, sorry, my original message was literally the classic terrible "it didn't work"

It was a dependency resolution/compatibility issue when running `npm install`

Reply:
@mraible 

Looks like vue is stuck at eslint v6 until the next version of vue v5?

https://github.com/vuejs/vue-cli/blob/v4/packages/%40vue/cli-plugin-eslint/package.json#L35

```txt
npm install

npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! Found: eslint@7.32.0
npm ERR! node_modules/eslint
npm ERR!   dev eslint@"^7.32.0" from the root project
npm ERR!   peer eslint@">= 4.12.1" from babel-eslint@10.1.0
npm ERR!   node_modules/babel-eslint
npm ERR!     dev babel-eslint@"^10.1.0" from the root project
npm ERR!   2 more (eslint-plugin-vue, vue-eslint-parser)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer eslint@">= 1.6.0 < 7.0.0" from @vue/cli-plugin-eslint@4.5.13
npm ERR! node_modules/@vue/cli-plugin-eslint
npm ERR!   dev @vue/cli-plugin-eslint@"~4.5.13" from the root project
```

My dependency fu isn't as strong in the JS world, any ideas? (all i did to repro was to revert the eslint version)

Reply:
It looks like you're using npm v7, that's why you see this issue. It doesn't happen for me with npm 6. This is likely because npm 6 didn't enforce peer dependencies like npm v7 does. You can workaround this by adding an `.npmrc` to the project and putting the following in it:

```
legacy-peer-deps = true
```

Of course, not upgrading to ESLint 7 works too!